src/shims/preact-router.js

```diff
@@ -0,0 +1,7 @@
+/** This file is a shim, aliased in for "preact-router" in the webpack config.
+ *  When components import 'preact-router', we want to give them back the copy
+ *  Zimbra passed down when it called the factory provided to zimlet().
+ */
+
+export const { route, Link, Match } = global.shims.preactRouter;
```

Comment:
`Match` is actually an export of `preact-router/match` not of `preact-router`.  Might want a separate alias/shim for that

Reply:
- [ ] remove `Match` and create alias for `preact-router/match` if desired